### DISCUSSION TITLE
READ_EXTERNAL_STORAGE & READ_MEDIA_AUDIO permissions

### DISCUSSION THREAD

**Discussion by BlackyHawky** [BlackyHawky/Clock#63] [Q&A] [answered]
@qw123wh, @odmfl, @Nilsu11:
Have you ever tried the application without the aforementioned authorizations?

On Android 6, 12 and 14, everything works fine and I don't see the point of having them in the manifest. 
Indeed, custom ringtone selection is still possible.

If you don't have the time to compile the application, you'll find attached a version without these permissions.

[Clock_2.5-debug.zip](https://github.com/user-attachments/files/15873871/Clock_2.5-debug.zip)

**Comment by Nilsu11** [comment, marked as answer]
There never was any requirement for this permission. It only was required for playing ringtones over Mediaplayer not over the ringtone class (which is used currently).
It works in my case.

**Comment by odmfl** [comment]
They are useful permissions if you can bring the "media alarm" functionality as in omniclock, currently they are not needed.  If we could bring that functionality here too it would be fantastic. @Nilsu11 You had started working on it, it would be nice if you could do it.

**Comment by BlackyHawky** [comment]
Thank you all for your answers. So I will remove these unnecessary authorizations from the manifest.
